harmonia: one more pass for bind volumes (#21582)
docker/docs: content/manuals/harmonia/_index.md

- KinD does not run on Project Harmonia due to some hard-coded assumptions to ensure it's running in a privileged container. K3d is a good alternative.
- Containers cannot access host through DNS `host.docker.internal`.
- File binds (non-directory binds) are currently static, meaning changes will not be reflected until the container is restarted. This also affects Compose configs and secrets directives.

> - Bind _mounts_, such as `-v /localpath:/incontainer` in the `docker run` command, require starting a file-sync.

Reviewer: s / starting / creating

> - Bind _mounts_, such as `-v /localpath:/incontainer` in the `docker run` command, require starting a file-sync.

Author: ```suggestion
- Bind _mounts_, such as `-v /localpath:/incontainer` in the `docker run` command, require creating a file-sync.
```

> - Starting a file-sync for a directory with a large amount of may take extra time to sync and become ready for us in a container.

Reviewer 2: ```suggestion
- Starting a file-sync for a directory with a large amount of may take extra time to sync and become ready for use in a container.
```

Reviewer: "a large amount *of data/files" may take..."

Reviewer: @stevvooe, the above suggestion didn't go in...

- Creating a synchronized file share for a directory with a large amount of [**missing noun**] may take extra time to sync and become ready for use in a container.

> - Starting a file-sync for a directory with a large amount of may take extra time to sync and become ready for use in a container.

Reviewer 3: ```suggestion
- Creating a [synchronized file share](/manuals/desktop/features/synchronized-file-sharing.md) for a directory with a large amount of may take extra time to sync and become ready for use in a container.
```

> - Bind _volumes_, such as those created with `docker volume create --driver local --opt type=none --opt o=bind --opt device=/some/host/path myvolname` or via the compose equivalent, are not supported.

Reviewer 3: nit: `local` is the default driver, so can be omitted in this example for brevity.

```suggestion
- Bind _volumes_, such as those created with `docker volume create --opt type=none --opt o=bind --opt device=/some/host/path myvolname` or via the compose equivalent, are not supported.
```

Author: I actually left this in so the context would be clear. Feel free to commit suggestion if you think its not necessary.
